resources/model/OpenStudio.idd

```diff
@@ -11945,19 +11945,23 @@ OS:AvailabilityManager:OptimumStart,
        \key ConstantStartTime
        \required-field
   N2,  \field Constant Temperature Gradient during Cooling
-       \units deltaC/hr
+       \units K/hr
```

Comment:
This is purely optional, `deltaC` would be replaced by `K` at https://github.com/NREL/OpenStudio/blob/d508c9f623fcc63a8a32e9bcb41ace29a4710ba8/src/utilities/units/IddUnitString.cpp#L56

Might as well just put `K/hr` to start with, so that it looks more in line with the `\ip-units R/h`